wasmer/instance.go

```diff
@@ -29,22 +29,27 @@ func (error *InstanceError) Error() string {
 // WebAssembly exported function. It is returned by `Instance`
 // functions only.
 type ExportedFunctionError struct {
-	functionName string
+	FunctionName string
```

Comment:
Why making it public?

Reply:
Because if the programmer is going to go through the trouble of type asserting an error into an ExportedFunctionError, that means that they want more detailed and parsed info than the string ExportedFunctionError.Error() returns. So why not make FunctionName accessible in this case?

While we're on the topic, I actually thing the error string for this error should be rewritten to do away with "Failed to call the `%s` exported function." because it is very a verbose way of explaining that a the call resulted in an error, which is obvious to the programmer already due to the context: i.e. I just got this after calling the exported function. 

Since function names may be dynamically programmed when calling an exported function, it makes sense to include the name in the error, but I'd rather see something like: `fmt.Sprintf("%q: %s", functionName, errStr)`, or `fmt.Errorf("%q: %w", functionName, err)`, if we are wrapping an error. That would print: `"sum": could not do the thing` type of error strings.

Reply:
I agree. Could you open an issue to address that so that we don't forget? Thanks!